server/utils/interventionsFormError.ts

```diff
@@ -1,25 +1,28 @@
-import { InterventionsServiceError, InterventionsServiceValidationError } from '../services/interventionsService'
+import createError from 'http-errors'
+import { InterventionsServiceValidationError } from '../services/interventionsService'
 import interventionsServiceErrorMessages from './interventionsServiceErrorMessages'
 import { FormValidationError } from './formValidationError'
 
 const dateFormFields = ['completion-deadline']
 
-// This re-throws the error if `interventionsServiceError` is not a validation error.
-export default function createFormValidationErrorOrRethrow(
-  interventionsServiceError: InterventionsServiceError
-): FormValidationError {
-  if (interventionsServiceError.validationErrors === undefined) {
-    throw interventionsServiceError
+// This re-throws the error if `err` is not an interventions service validation error.
+export default function createFormValidationErrorOrRethrow(err: Error): FormValidationError {
+  if (createError.isHttpError(err)) {
+    if (err.response?.body?.validationErrors) {
```

Comment:
is it worth wrapping this in a private method/variable e.g. `isInterventionsServiceValidationError` to make it a bit more obvious what this condition is? `err.response?.body?.validationErrors` doesn't quite make it clear enough I think.

Reply:
yeh that's a good idea.